services/updateStatus.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
yar permission don't know we are doing this permission stuff ek or jaga per b jaha we are checking k jo banda profile dekh raha hai wo khud hai k nai if yes then allow him to edit otherwise don't allow him to edit